### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LindsayYoung**
[Link to ZenHub issue.](link-goes-here)

## Checklist:

_these will eventually be enforced by CircleCI; we need to set that up at the USDOJ org level_

+ [ ] If front end or functionality change, run locally and check http://0.0.0.0:8000/report/.
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests)

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
@ilodidoj would you like to review this one?

File: README.md [hunk 1]
```diff
@@ -79,20 +79,22 @@ You only need to get the services stood up and configure the S3 bucket once.
 
 For working with cloud.gov directly, you will need to [install the cloud foundry cli](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html). That will allow you to run the `cf` commands in a terminal.
 
-Log on with `cf login -a api.fr.cloud.gov --sso` and go to the link to sign in and get your pass-code.
+First, login to cloud.gov at https://login.fr.cloud.gov/login and then, get a passcode https://login.fr.cloud.gov/passcode.
+
+Log on with `cf login -a api.fr.cloud.gov --sso-passcode <put_passcode_here>` and go to the link to sign in and get your pass-code.
```
Comment (alexsoble) [inline README.md]: Seems like this passcode link instruction is repeated above, unless I'm missing something?

Comment (LindsayYoung) [reply README.md]: the instruction above is in a browser the second line is in the command line

Comment (LindsayYoung) [reply README.md]: I see what I was missing - I didn't remove the second log in

Comment (LindsayYoung) [reply README.md]: I see I what I was missing! I need to take out the second login lines since you have to do it before for the passcode variable

**Comment by alexsoble** [comment]
@LindsayYoung Would you like me to review and merge, or should we wait for @ilodidoj review?

**Pull request merged**